packages/react/src/components/DataTable/Table.js

```diff
@@ -36,9 +36,11 @@ export const Table = ({
     [`${prefix}--data-table--visible-overflow-menu`]: !overflowMenuOnHover,
   });
   const table = (
-    <table {...other} className={componentClass}>
-      {children}
-    </table>
+    <div className={`${prefix}--data-table-content`}>
```

Comment:
We may need to look into how the toolbar renders on smaller screens as well...

![Screen Shot 2020-07-28 at 10 33 53 AM](https://user-images.githubusercontent.com/11928039/88700853-143b0f80-d0be-11ea-9878-f234be1c0d04.png)

Reply:
@tw15egan would you like me to look into that or should I create a separate issue for this? It seems unrelated to the changes I made unless I am mistaken.

Reply:
Applied a temporary fix in this PR to make it scroll when there is not enough space. We could do the same trick as in the pagination to hide text that is not needed but that is more of a UX question.

Reply:
This change looks great 👍

Reply:
Noticed in Percy we're also having this issue with the `with toolbar` version on smaller breakpoints. Wondering if it makes sense that container scrolls as well? 

<img width="910" alt="Screen Shot 2020-07-29 at 9 45 55 AM" src="https://user-images.githubusercontent.com/11928039/88828989-d8b54980-d180-11ea-99a6-07ed83ce3fc0.png">

We can track that in a separate issue if you'd like, but just wanted to point it out if you had a quick fix!